### ISSUE TITLE
iOS VoiceOver reads the original selectbox as well

### ISSUE THREAD

**Issue by ciiqq** [snapappointments/bootstrap-select#1483]
When using the one-finger swipe gesture (the "tab" gesture) on the generated select box, VoiceOver focuses on the hidden, opacity: 0 negative tabindex original select element. The user cannot interact with it, but it causes confusion. This only happens on iOS VoiceOver, not OSX VoiceOver or Windows NVDA. You can try it out at the Examples page: https://silviomoreto.github.io/bootstrap-select/examples/

Happens both on iPhone 5s and iPad Air

**Comment by NicolasCARPi** [comment]
Is this still an issue?
